bciers/apps/administration/app/components/datagrid/ActionCellFactory.tsx

```diff
@@ -1,29 +1,27 @@
 import Link from "next/link";
+import { GridRenderCellParams } from "@mui/x-data-grid";
 
 const ActionCellFactory = ({
-  href,
+  generateHref,
   replace,
-  className,
   cellText,
 }: {
-  href: string;
+  generateHref: (params: GridRenderCellParams) => string;
   replace: boolean;
-  className: string;
   cellText: string;
 }) => {
-  return () => {
-    const renderCell = () => {
-      return (
-        <div>
-          <Link className={className} href={href} replace={replace}>
-            {cellText}
-          </Link>
-        </div>
-      );
-    };
-
-    return renderCell;
+  const renderCell = (params: GridRenderCellParams) => {
+    return (
+      <Link
+        className="no-underline text-bc-link-blue whitespace-normal"
+        href={generateHref(params)}
+        replace={replace}
+      >
+        {cellText}
+      </Link>
+    );
```

Comment:
@Sepehr-Sobhani thanks for this, I borrowed it from your PR and it's working super well 😍 

 What do you think about keeping the styles in this for consistency? I don't think we should have to repeat the same styling every time we use it.  
  
If there is a good reason to want to customize styling we can pass in `className` and add it to the end of the classes, which will allow users to overwrite the default styles:  
`className={`no-underline text-bc-link-blue whitespace-normal ${className}`}

Reply:
@marcellmueller 
Is your "borrowed" code from https://github.com/bcgov/cas-registration/blob/chore/1608-facility-form-edit/bciers/libs/components/src/datagrid/cells/ActionCellFactory.tsx ?
If yes, and you like it; perhaps, put your `bciers/apps/administration/app/components/datagrid/ActionCellFactory.tsx` in the `libs` folder as I did...along with your beauty improvement suggestion about the styles.

Reply:
Haha I didn't even notice that, thanks!

Reply:
Yes, I think it's good to have a way to customize reusable components, just in case.